Cargo.toml

```diff
@@ -46,8 +46,12 @@ filetime = "0.2.10"
 
 [features]
 default = ["native-roots"]
+# Use Rustls with native roots
 native-roots = ["reqwest/rustls-tls-native-roots"]
+# Use Rustls with WebPK roots
 webpki-roots = ["reqwest/rustls-tls-webpki-roots"]
+# Use native TLS (SChannel on Windows, Secure Transport on macOS and OpenSSL otherwise)
```

Comment:
Here I am not sure, instinctively, I would say that
a) this comment can only become outdated
b) if you are compiling for a platform that doesn't support the other options, you probably know what this means
c) if you don't know what it means and are unaware what this means, you are probably not looking into `Cargo.toml` in the first place

If we want to bring attention to these features, we could have a small section in the README or documentation about building, here I don't see much of a benefit.

Reply:
The idea was to highlight/explain the three different options to handle TLS:

- RusTLS with WebPKI roots
- RusTLS with native roots
- Native TLS

I'm not sure if this belongs in the docs, since configuring this is rather niche, and people who want to customize the build process probably look at `Config.toml` anyways. But maybe a single intro section would be better, than a line above every option.